### ISSUE TITLE
python / matlab versions

### ISSUE THREAD

**Issue by shyzaks** [RocketDataScientist/DataHack-2017#4]
Hi
you stated that you used certain version to run your code.
Can we use in our work different versions?
(To the best of my understanding the answer is "yes" since only the results matter.)
thanks
Shay

**Comment by RocketDataScientist** [comment]
Indeed, no problem. We are language agnostic, so you can even use Fortran or Assembly to make things more interesting :-)
